Library/Homebrew/cmd/commands.rb

```diff
@@ -8,51 +8,66 @@ module Homebrew
   module_function
 
   def commands
-    if ARGV.include? "--quiet"
-      cmds = internal_commands + external_commands
-      cmds += internal_developer_commands
-      cmds += HOMEBREW_INTERNAL_COMMAND_ALIASES.keys if ARGV.include? "--include-aliases"
-      puts Formatter.columns(cmds.sort)
-    else
-      # Find commands in Homebrew/cmd
-      puts "Built-in commands"
-      puts Formatter.columns(internal_commands)
-
-      # Find commands in Homebrew/dev-cmd
-      puts
-      puts "Built-in developer commands"
-      puts Formatter.columns(internal_developer_commands)
-
-      # Find commands in the path
-      unless (exts = external_commands).empty?
+    command = CommandsCommand.new
+    command.commands
+  end
+
+  class CommandsCommand < CommandOptions
+    def initialize
+      super
+      option "--quiet", "List only the names of commands without the header"
+      option "--include-aliases", "The aliases of internal commands will be included"
+    end
+
+    def commands
```

Comment:
Calling `commands` on a `CommandsCommand` instance seems a bit redundant, and it'll have to be different for every command — how about something generic like `call` — which is the standard name in Ruby for objects that perform a specific task?